README.md

```diff
@@ -211,6 +212,7 @@ Please take a quick look at the [contribution guidelines](CONTRIBUTING.md) first
 * [get_it_mixin](https://pub.dev/packages/get_it_mixin) - mixin on Stateless- and StatefulWidgts that makes binding the Widget to Objects stored in GetIt easier. Makes get_it to a full state management solutio
 * [mobx](https://pub.dev/packages/mobx) - MobX is a state-management library that makes it simple to connect the reactive data of your application with the UI.
 * [provider](https://pub.dev/packages/provider) - A wrapper around InheritedWidget to make them easier to use and more reusable.
+* [getX](https://pub.dev/packages/get) - GetX is an extra-light and powerful solution for Flutter. It combines high performance state management, intelligent dependency injection, and route management in a quick and practical way.
```

Comment:
As described in the contribution.md could you please insert the package in alphabetic order?

Reply:
Ok I will make it